src/lib/notificationEmitter.js

```diff
@@ -24,42 +24,18 @@
 
 const io = require('./api-server').socketIO
 
-// io.on("connection", socket => {
-//   // socket.leave(socketId);
-//   // socket.join(socketId);
-//   //   socket.to(doc.id).emit("document", doc);
-//   io.emit("logRefresh", []);
-//   const sendNotification = counter => {
-//     socket.emit("newLog", { message: 'Log no'+counter});
-//     setTimeout(sendNotification,1000,counter+1);
-//   };
-//   sendNotification(1);
-// });
-
-const broadcastLog = (log, sessionID = null) => {
-  io.emit('newLog', {
+const broadcastLog = (log, sessionID) => {
+  io.emit(`newLog${sessionID ? `/${sessionID}` : ''}`, {
     logTime: new Date(),
     ...log
   })
-  if (sessionID) {
-    io.emit('newLog/' + sessionID, {
-      logTime: new Date(),
-      ...log
-    })
-  }
```

Comment:
broadcastLog should emit two events if we call it with two arguments (sessionid). I think your change here is breaking original functionality.